docs/UserGuide.md

```diff
@@ -112,21 +115,21 @@ This practice helps prevent data loss and ensures a proper shutdown of the appli
 5. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
```

Comment:
for step 4, mb can change it to, "use cd to change the current working directory into the folder you put the jar file in..." instead of jus 'cd into the folder...'

Reply:
Changed step 4. Thanks for the suggestion